src/views/services/tyeApplicationNode.ts

```diff
@@ -10,7 +10,7 @@ import TyeServiceNode from './tyeServiceNode';
 import { TyeClientProvider } from '../../services/tyeClient';
 import ext from '../../ext';
 
-export class TyeApplicationNode implements TreeNode {
+export default class TyeApplicationNode implements TreeNode {
```

Comment:
Just curious, what is the reasoning behind making this a default export?

Reply:
Mostly to match how the other tree nodes were exported.  (I usually try to make the "main" thing in a file the default export (e.g. the implementation), with other related things (e.g. related interfaces) as explicit exports.